livre/Lib/Configuration/ApplicationParameters.cpp

```diff
@@ -37,83 +38,56 @@ std::istream& operator>>( std::istream& is, vmml::vector<2ul, int>& vec )
 namespace livre
 {
 
-const std::string ANIMATIONENABLED_PARAM = "animation";
+const std::string ANIMATION_PARAM = "animation";
 const std::string FRAMES_PARAM = "frames";
-const std::string DATAFILE_PARAM = "volume";
 const std::string NUMFRAMES_PARAM = "num-frames";
 const std::string CAMERAPOS_PARAM = "camera-position";
 const std::string SYNC_CAMERA_PARAM = "sync-camera";
 
 ApplicationParameters::ApplicationParameters()
-    : Parameters( "Application Parameters" )
-    , animationEnabled( false )
-    , frames( Vector2i( 0, std::numeric_limits<int>::max( )))
+    : cameraPosition( Vector3f( 0, 0, -2.0 ))
+    , frames( Vector2ui( 0, std::numeric_limits<Vector2ui::value_type>::max( )))
+    , maxFrames( 0xFFFFFFFFu )
+    , animation( 0 )
     , isResident( false )
-    , cameraPosition( Vector3f( 0, 0, -2.0 ))
     , syncCamera( false )
 {
-    configuration_.addDescription( configGroupName_, DATAFILE_PARAM,
-                                   "URI of volume data source", dataFileName );
-    configuration_.addDescription( configGroupName_, ANIMATIONENABLED_PARAM,
-                                   "Enable animation mode", animationEnabled );
+    configuration_.addDescription( configGroupName_, ANIMATION_PARAM,
+                                   "Enable animation mode", animation, 1 );
```

Comment:
give more descriptive information what the parameters means, e.g. negative value means backwards etc.